federation/deploy/init/init.go

```diff
@@ -0,0 +1,573 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// This program implements the initialization code required to start a
+// federation API server. This code is intended to be run inside an init
+// container in the federation API server pod.
```

Comment:
If there is some documentation of init container somewhere, will be great if you can link to it here.